app/src/main/kotlin/io/homeassistant/companion/android/settings/controls/views/ManageControlsView.kt

```diff
@@ -119,6 +124,25 @@ fun ManageControlsView(
         }
 
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.UPSIDE_DOWN_CAKE || !panelEnabled) {
+            if (serversList.size > 1) {
+                item {
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.padding(start = 16.dp, bottom = 16.dp)
```

Comment:
I feel like this also needs end padding; your screenshot shows the toggle almost touching the right edge of the screen.

Reply:
Good point. It would also have been an issue on smaller devices as well, so I've added some padding (and fixed some issues with the text wrapping):
![Screenshot_20250412-101247](https://github.com/user-attachments/assets/385ea011-3f83-4b4c-9402-75590a42e4a9)